apps/prairielearn/src/lib/chunks.ts

```diff
@@ -685,7 +684,14 @@ module.exports.getChunksDirectoriesForCourseId = (courseId) => {
     chunks: path.join(baseDirectory, '__chunks', 'chunks'),
     unpacked: path.join(baseDirectory, '__chunks', 'unpacked'),
   };
-};
+}
+
+interface CourseWithRuntimeDirectory {
+  /** The database ID of the course. */
+  id: string;
+  /** The path to the course source (not the chunks) */
+  path: string;
+}
```

Comment:
Eventually we'll want to pick these off a `Course` model, but this is good enough for now.

Reply:
Yep, I like this strategy of "first make it TS at all, then switch to clean DB-based models later".